birdhouse/scripts/bootstrap-testdata

```diff
@@ -10,8 +10,22 @@
 # Need write-access to DATASET_ROOT (/data/datasets/).
 
 
+THIS_FILE="$(readlink -f "$0" || realpath "$0")"
+THIS_DIR="$(dirname "${THIS_FILE}")"
+COMPOSE_DIR="${COMPOSE_DIR:-$(dirname "${THIS_DIR}")}"
+
+if [ -f "${COMPOSE_DIR}/read-configs.include.sh" ]; then
+    . "${COMPOSE_DIR}/read-configs.include.sh"
+
+    # Get THREDDS_SERVICE_DATA_LOCATION_ON_HOST
+    read_configs
+fi
+
+
 if [ -z "${DATASET_ROOT}" ]; then
-    DATASET_ROOT="${BIRDHOUSE_DATA_PERSIST_ROOT}/${THREDDS_SERVICE_DATA_LOCATION_ON_HOST}"
+    # Default for when unable to source read-configs.include.sh (ie when
+    # used standalone outside of the checkout).
+    DATASET_ROOT="${THREDDS_SERVICE_DATA_LOCATION_ON_HOST:=/data/datasets}"
```

Comment:
This should be `:-` instead of `:=`

Reply:
This will set `THREDDS_SERVICE_DATA_LOCATION_ON_HOST` if undefined, and then set `DATASET_ROOT` with it, which will result in the same behavior in this case.

Since the rest of the script doesn't rely on `THREDDS_SERVICE_DATA_LOCATION_ON_HOST`, not a big deal, but we can use `:-` since it is more common to indicate the default value.

However, maybe consider `BIRDHOUSE_DATA_PERSIST_ROOT` as well:

```sh
DATASET_ROOT="${THREDDS_SERVICE_DATA_LOCATION_ON_HOST:-${BIRDHOUSE_DATA_PERSIST_ROOT:-/data}/datasets}"
```

Reply:
Yes I started to use `:=` instead of `:-` to actually set `THREDDS_SERVICE_DATA_LOCATION_ON_HOST` because had the sourcing of `read-configs.include.sh` worked, that `THREDDS_SERVICE_DATA_LOCATION_ON_HOST` would also exist for real.

I was trying to emulate the successful sourcing of `read-configs.include.sh` even in standalone mode, but it was not needed since we don't use `THREDDS_SERVICE_DATA_LOCATION_ON_HOST` afterwards.  I can revert to `:-` no problem.